tests/unittests/tests-hacl/tests-hacl.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2018 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     unittests
+ * @{
+ *
+ * @file
+ * @brief       HACL* crypto library tests
+ *
+ * @author      Benjamin Beurdouche <benjamin.beurdouche@inria.fr>
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ * @author      Martin Landsmann <Martin.Landsmann@HAW-Hamburg.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
```

Comment:
Not completely sure on this include, but it looks unnecessary to me as you're not doing actual IO ops